include/swift/AST/DiagnosticsSema.def

```diff
@@ -1499,7 +1499,7 @@ ERROR(access_control_extension_more,none,
       "be declared %select{%error|fileprivate|internal|public|%error}2",
       (AccessLevel, DescriptiveDeclKind, AccessLevel))
 ERROR(access_control_extension_open,none,
-      "extensions cannot use 'open' as their default access; use 'public'",
+      "extensions cannot use 'open' as their default access; remove the default access and mark methods as 'open' instead",
```

Comment:
I don't have specific suggestions for better wording at the moment, but specifying "methods" is not quite right. This applies to properties too. Similarly, the code in `TypeCheckAttr` is only checking for `FuncDecl` but it needs to account for properties too.

Reply:
Thanks for the input! I made it so that diagnostics are produced for `VarDecl` as well. And I changed the wording to `extensions cannot use 'open' as their default access; remove the default access and mark members as 'open' instead`. What do you think of using `member` in lieu of `methods/properties`?